CMakeLists.txt

```diff
@@ -92,7 +109,7 @@ endforeach()
 
 # =================================================================================================
 # OPTIONS
-
+option(BUILD_SHARED_LIBS "Build shared library" OFF)
```

Comment:
This one is already set below, with the default value `ON`. Maybe worth separating out CMake options and CP2K-specific options for clarity?